install/terraform/modules/helm/helm.tf

```diff
@@ -85,8 +85,8 @@ locals {
 resource "helm_release" "agones" {
   name         = "agones"
   force_update = "true"
-  repository   = "${data.helm_repository.agones.metadata.0.name}"
-  chart        = "${var.chart}"
+  repository   = data.helm_repository.agones.metadata.0.name
+  chart        = var.chart
   timeout      = 420
 
   # Use terraform of the latest >=0.12 version
```

Comment:
"${length(var.values_file) == 0 ? "" : file("${var.values_file}")}",
This is where `values_file` would be used